frontend/src/i18n/locales/zh-cn.json

```
@@ -71,6 +71,7 @@
     "total": "总额",
     "copy-payment-link": "复制付款链接",
     "enter-an-amount": "请输入金额",
+    "enter-a-recipient": "請輸入收件者",
```

```suggestion
    "enter-a-recipient": "请输入收款人",
```
 Hi Alex, just changed these to simplified Chinese.